Angular/src/app/edit-role-modal/edit-role-modal.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Edit Role</h5>
+    <button type="button" class="close" aria-label="Close" (click)="activeModal.dismiss()">
+      x
+    </button>
+  </div>
+  
+  <form [formGroup]="editRoleForm" (ngSubmit)="onSubmit()" autocomplete="off">
+    <div class="modal-body" data-keyboard="false" data-backdrop="static">
+      <div *ngIf="roleExists" class="error-message">There is already an existing role for the band/capability you have
+        specified</div>
+      <div class="form-group">
+        <label for="roleName">Role Name</label>
+        <input type="text" formControlName="roleName" class="form-control"
+          [ngClass]="{ 'is-invalid': submitted && formControls.roleName.errors }" />
+        <div *ngIf="submitted && formControls.roleName.errors" class="invalid-feedback">
+          <div *ngIf="formControls.roleName.errors.required">
+            Role Name is a required field
+          </div>
+        </div>
+      </div>
+  
+      <div class="form-group">
+        <label for="roleSummary">Role Summary</label>
+        <textarea type="text" formControlName="roleSummary" class="form-control" rows="3" maxlength="1000" 
+         [ngClass]="{'is-invalid': submitted && formControls.roleSummary.errors}" required></textarea>
+        <div *ngIf="submitted && formControls.roleSummary.errors" class="invalid-feedback">
+          <div *ngIf="formControls.roleSummary.errors.maxlength">
+            Max characters reached for Role Summary
```

Comment:
You can make your validation error messages more specific in this file (e.g. here, `Role Summary should be 1000 characters max`)